plugins/ci/commands/query-job-status.md

```
@@ -76,27 +91,34 @@ The command performs the following steps:
      "gcs_path": "gs://origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.14-e2e-aws-ovn/1234567890"
    }
    ```
+   Claude derives and displays: `https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-openshift-release-master-ci-4.14-e2e-aws-ovn/1234567890`
 
-2. **Check running job**:
+2. **Query a recently triggered job (auto-polling)**:
    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)